azure-pipelines.yml

```diff
@@ -16,7 +16,11 @@ steps:
     displayName: 'Runs linting checks'
 
   - script: |
-      npm test -- --coverage
```

Comment:
We do not need to remove the previous tests. Since the tests are ran against other functions different from accessibility tests.
What we could do is define the conditions necessary for us to run accessibility tests so that we do not run an expensive test on every commit. Maybe a merge to dev / master